lib/ramble/ramble/workspace/workspace.py

```diff
@@ -1688,6 +1709,14 @@ def get_workspace_env_vars(self):
             if namespace.env_var in \
             workspace_dict[namespace.ramble] else None
 
+    def get_workspace_internals(self):
```

Comment:
these `get_workspace_xxx` blocks are very similar, do you think we should abstract the out ?

Reply:
Done!